ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-file-answer/activityFileAnswer.component.ts

```diff
@@ -71,47 +70,38 @@ export class ActivityFileAnswer implements OnInit, OnDestroy {
                     }),
                     takeUntil(this.ngUnsubscribe)
                 )
-                .subscribe((res: FileUploadResponse) => {
-                    this.submitFileUpload(file, res.uploadGuid, res.uploadUrl);
+                .subscribe((res: FileUploadResponse[]) => {
+                    this.validateAndSubmitFileUpload(filesGroup, res);
                 });
         }
     }
 
-    submitFileUpload(file: File, uploadGuid: string, uploadUrl: string): void {
-        const failedLocalValidator = this.getFailedLocalValidator(file);
-        if (failedLocalValidator) {
-            this.errorMessage = failedLocalValidator.result as string;
-            this.componentBusy.emit(false);
-            return;
-        }
+    validateAndSubmitFileUpload(files: File[], fileUploadRes: FileUploadResponse[]): void {
```

Comment:
Seems like it may be a private method.

Reply:
Thank you. Made private and moved below (closer to private methods).